store/index.js

```diff
@@ -212,5 +212,5 @@ export default function configureStore(initialState) {
         detectNetwork: detect,
     };
 
-    return configureServiceStore({}, appReducer, offlineOptions, getAppReducer, {enableBuffer: false});
+    return configureServiceStore(initialState, appReducer, offlineOptions, getAppReducer, {enableBuffer: false});
```

Comment:
For some reason, we never passed the preloaded state into the store before. This is only provided by test code, so it'll only affect tests using the store in this way. I'm surprised this wasn't done before, but I guess webapp tests are usually written using more mocks and less relying on real state.

Reply:
I guess rehydration somehow fixes this for us?

Reply:
What does this solve? Just curious.

Reply:
This doesn't have anything to do with rehydration or saving redux state in the webapp. `initialState` is always an empty object when you're running the app itself, but it can be passed down with a set value for use in unit tests. A lot of the tests I've written in mattermost-redux rely on being able to set up an initial state in the store